ddtrace/internal/datadog/profiling/releasenotes/notes/fix-profiling-v2-trunc-c442c864bcfad7ad.yaml

```diff
@@ -0,0 +1,5 @@
+---
+fixes:
+  - |
+    Fixes an issue where Profiling stack v2 would erroneously emplace informational pseudo-frames
+    on the leaf instead of the root of the callstack.
```

Comment:
```suggestion
    profiling: Fixes an issue where stack v2 would erroneously emplace informational pseudo-frames
    on the leaf instead of the root of the callstack.
```